src/model/comment.ml

```diff
@@ -147,20 +147,16 @@ let find_zero_heading docs : link_content option =
     docs
 
 let extract_frontmatter docs : _ =
-  let parse_frontmatter s =
-    let lines = Astring.String.cuts ~sep:"\n" s in
-    List.filter_map (fun line -> Astring.String.cut ~sep:":" line) lines
-  in
   let fm, content =
     let fm, rev_content =
       List.fold_left
         (fun (fm_acc, content_acc) doc ->
           match doc.Location_.value with
           | `Code_block (Some "meta", content, None) ->
-              (parse_frontmatter content.Location_.value :: fm_acc, content_acc)
+              (content.Location_.value :: fm_acc, content_acc)
```

Comment:
The previous code was more logical (parse each blocks and concatenate the values rather than concatenate the blocks and parse something no one wrote) and allowed more graceful handling of errors with locations.
Anyway, that's a temporary syntax.

Reply:
Indeed, we are going to have the frontmatter consistently at the top, so there won't be any distinction.
So, I suggest leaving it as is.